source/session.js

```diff
@@ -304,15 +300,16 @@ export class Session {
    * @param  {*} data  The data to decode.
    * @return {*}      Decoded data
    */
+
   decode(data, identityMap = {}) {
-    if (data == null) {
-      return data;
```

Comment:
I removed it. The rest of the checks are not picking up null, and the "default" return is also `data`. That, combined with it being loose equality and thus probably subject to some kind of weird quirks that I didn't feel like googling, made me remove it.

Reply:
Loose equality for null typically is used for null or undefined:

```
> "null == null:	" true
> "undefined == null:	" true
> "0 == null:		" false
> "false == null:	" false
```

Since undefined is not valid JSON (which is what is being decoded), this will only be null.  null fails the checks, and is returned as passed in as before. All good!